Obsidian.API/_Interfaces/IWorld.cs

```diff
@@ -7,5 +10,8 @@ public interface IWorld
 
         public long Time { get; }
         public Gamemode GameType { get; }
+
+        public Task SpawnExperienceOrb(PositionF position, short count);
```

Comment:
What does `count` mean? The variable name is unclear and misses XML docs for clarification.

Reply:
I'm pretty sure that it's the experience orb count. Maybe the method should be renamed to `SpawnExperienceOrbs`.

Reply:
A rename of the method to more clearly define that it's multiple orbs would be nice, yes.